### DISCUSSION TITLE
Adding quantities to recipe, so it can be indicated in the shopping list

### DISCUSSION THREAD

**Discussion by Launster** [TomBursch/kitchenowl#531] [Ideas]
Hello, I am quite new on this project, starting to play with it features.

I am a little surprised that there is no information about the amount of ingredient in the recipes or in the shopping list. 
This is either me missing something, or the app (this is my personnal point of view of course)

This is important for the recipe (if you need 300g of sth, this is not the same as 500g) and for the shopping list (same, and if I got several recipe with the same ingredient, I do not want to buy too less)

**Comment by TomBursch** [comment]
Hi, you can add quantities in the description field. If you search for an item `300g Sugar` the 300g should be put into the description field automatically. Alternatively you can separate the description using a `,` and everything following it will be put in the description. Descriptions are scaled automatically when you increase the servings of a recipe.

https://docs.kitchenowl.org/latest/Tips-%26-Tricks/

**Reply by remyayad** [reply]
Thank you for your response! However, it doesn’t seem to work in all cases. For instance, if I wanted to add 6 slices of cooked and chopped bacon, how would I do that? Also, would it scale automatically as I adjust the number of servings?

**Reply by crosspolar** [reply]
I hope I get you right, but you would add it to the recipe like this
![grafik](https://github.com/user-attachments/assets/0ba522b9-ddd7-44ef-b5ef-e99401eb6609)
Don't forget to set also the amount of portions this is for in the recipe. When planning your meal, you can scale it and it will be transformed
![grafik](https://github.com/user-attachments/assets/acae4130-d607-42d0-bb1a-2840ff3f8658)

**Comment by damitjimii** [comment]
I’d also like to see quantities as amount and unit (i.e. 190 g or 1.5 cups) this should make scaling and shopping lists a lot easier.  Additionally; I’d also love to see a volume measurement to weight conversion button or a display option of unit type.    Great stuff so far; love to see how the app progresses.

**Comment by cybrengineer** [comment]
Same here, the app is awesome. But I would also like to see the addition of more units outside of just grams, something like cups, teaspoons, tablespoons etc
